deeppavlov/configs/kbqa/kbqa_rus.json

```diff
@@ -11,199 +11,51 @@
     "in_y": ["y"],
     "pipe": [
       {
-        "id": "classes_vocab",
-        "class_name": "simple_vocab",
-        "fit_on": ["y"],
-        "save_path": "{MODEL_PATH}/relation_prediction/classes.dict",
-        "load_path": "{MODEL_PATH}/relation_prediction/classes.dict",
-        "in": "y",
-        "out": "y_ids"
-      },
-      {
-        "in": "x",
-        "out": "x_tok",
-        "id": "my_tokenizer",
-        "class_name": "nltk_tokenizer",
-        "tokenizer": "wordpunct_tokenize"
-      },
-      {
-        "in": ["x_tok"],
-        "class_name": "str_lower",
-        "out": ["x_lower"]
-      },
-      {
-        "in": "x_lower",
-        "out": "x_emb",
-        "id": "my_embedder",
-        "class_name": "fasttext",
-        "load_path": "{DOWNLOADS_PATH}/embeddings/ft_native_300_ru_wiki_lenta_nltk_word_tokenize.bin"
-      },
-      {
-        "in": "y_ids",
-        "out": "y_onehot",
-        "class_name": "one_hotter",
-        "depth": "#classes_vocab.len"
-      },
-      {
-        "in": ["x_emb"],
-        "in_y": ["y_onehot"],
-        "out": ["y_pred_probas"],
-        "main": true,
-        "class_name": "keras_classification_model",
-        "save_path": "{MODEL_PATH}/relation_prediction/model",
-        "load_path": "{MODEL_PATH}/relation_prediction/model",
-        "embedding_size": "#my_embedder.dim",
-        "n_classes": "#classes_vocab.len",
-        "kernel_sizes_cnn": [
-          1,
-          2,
-          3
-        ],
-        "filters_cnn": 256,
-        "optimizer": "Adam",
-        "learning_rate": 0.05,
-        "learning_rate_decay": 0.1,
-        "loss": "categorical_crossentropy",
-        "last_layer_activation": "softmax",
-        "text_size": 36,
-        "coef_reg_cnn": 1e-3,
-        "coef_reg_den": 1e-2,
-        "dropout_rate": 0.5,
-        "dense_size": 300,
-        "model_name": "cnn_model"
-      },
-      {
-        "in": "y_pred_probas",
-        "out": "y_pred_ids",
-        "class_name": "proba2labels",
-        "max_proba": true
-      },
-      {
+        "config_path": "{CONFIGS_PATH}/classifiers/relation_prediction_rus.json",
```

Comment:
This file does not exist, it crashes all the tests.

Reply:
I pushed relation_prediction_rus.json config.